scripts/ci_build_and_test_in_container.sh

```diff
@@ -207,6 +219,16 @@ else
   fi
 fi
 
+if [[ ! -z "${SCCACHE_CREDS}" ]]; then
+  echo "sccache post-build state"
+  or_die ${SCCACHE} --show-adv-stats
+fi
+
+if [[ "${CODE_COVERAGE}" = true ]]; then
+  or_die ninja coreComponents_coverage
+  cp -r ${GEOSX_BUILD_DIR}/coreComponents_coverage.info.cleaned /tmp/geos/geos_coverage.info.cleaned
```

Comment:
```suggestion
  cp -r ${GEOSX_BUILD_DIR}/coreComponents_coverage.info.cleaned ${GEOS_SRC_DIR}/geos_coverage.info.cleaned
```
?

Or maybe using `DATA_EXCHANGE_DIR` if it does not interfere with the integrated tests (not sure).

Reply:
I meant to do this but forgot. thanks.